native/explorer/src/dataframe.rs

```diff
@@ -455,6 +455,35 @@ pub fn df_from_series(columns: Vec<ExSeries>) -> Result<ExDataFrame, ExplorerErr
     Ok(ExDataFrame::new(df))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn df_relocate(
+    df: ExDataFrame,
+    columns: Vec<&str>,
+    position: u64,
+) -> Result<ExDataFrame, ExplorerError> {
+    let mut first_series = df.get_columns().to_owned();
+    let second_series = first_series.split_off(position as usize);
+
+    let (first_series, first_to_relocate): (Vec<Series>, Vec<Series>) = first_series
+        .into_iter()
+        .partition(|series| !columns.contains(&series.name()));
+
+    let (second_series, second_to_relocate): (Vec<Series>, Vec<Series>) = second_series
+        .into_iter()
+        .partition(|series| !columns.contains(&series.name()));
+
+    let series = first_series
+        .into_iter()
+        .chain(first_to_relocate.into_iter())
+        .chain(second_to_relocate.into_iter())
```

Comment:
This may be wrong, actually. This wouldn't re-insert the columns in the correct order
```elixir
df = Explorer.DataFrame.new(col_a: [1, 2], col_b: [5.1, 5.2], col_c: [4,5], col_d: ["yes", "no"], col_e: [4, 1])
#  .. 
df2 = Explorer.DataFrame.relocate(df, [4, 0], before: 2)
> #Explorer.DataFrame<
  Polars[2 x 5]
  col_b float [5.1, 5.2]
  col_e integer [4, 1]
  col_a integer [1, 2]
  col_c integer [4, 5]
  col_d string ["yes", "no"]
>
Explorer.DataFrame.dump_csv(df2)
> {:ok, "col_b,col_a,col_e,col_c,col_d\n5.1,1,4,4,yes\n5.2,2,1,5,no\n"}
```
So while the implementation in Elixir orders it appropriately, it isn't ordered correctly here for the underlying dataframe

Reply:
Okay, should be addressed in 234c61c1d551c3146b766dd7b810b46017aedf84 :)